src/components/pool/branded-redirect/constants.ts

```diff
@@ -0,0 +1,32 @@
+import { networkId } from '@/composables/useNetwork';
+import { Network } from '@balancer-labs/sdk';
+
+interface BrandedRedirectData {
+  id: string;
+  title: string;
+  description: string;
+  btnText: string;
+  link: string;
+}
+
+type BrandedRedirects = Record<string, BrandedRedirectData>;
+
+const POLYGON_BRANDED_REDIRECTS: BrandedRedirects = {
+  '0x726e324c29a1e49309672b244bdc4ff62a270407000200000000000000000702': {
+    id: 'xave',
+    title: 'brandedRedirect.xave.title',
+    description: 'brandedRedirect.xave.description',
+    btnText: 'brandedRedirect.xave.btnText',
+    link: 'https://xave.co/',
+  },
```

Comment:
It's likely that we will end up with a lot of cases where there is many pools under a single brand. So ideally we map the `BrandedRedirectData` to a key, something like:
```suggestion
  'xave': {
    id: 'xave',
    title: 'brandedRedirect.xave.title',
    description: 'brandedRedirect.xave.description',
    btnText: 'brandedRedirect.xave.btnText',
    link: 'https://xave.co/',
  },
```
And then elsewhere we map all the relevant pool IDs to that key, e.g.
```
'xave': ['0x726e324c29a1e49309672b244bdc4ff62a270407000200000000000000000702']
```

Reply:
this file now contains only brand data. 
pools that should have branded redirect are only here `src/constants/pools.ts` in `BrandedRedirect` field